package.json

```diff
@@ -28,7 +29,7 @@
   },
   "devDependencies": {
     "moment": "2.14.1",
```

Comment:
I believe it is due to changing internal APIs - https://github.com/moment/moment/blob/develop/CHANGELOG.md#290-see-full-changelog - the deprecation of `moment.fn.zone`